frontend/src/App.tsx

```diff
@@ -34,18 +36,23 @@ function App() {
   if (userProfile.isSignedIn()) {
     const user = userProfile.getProfile();
 
-    const defaultStateFilter = []
+    const defaultStatementStateFilter = []
+    const defaultSentenceStateFilter = [
+      ComposerSentenceListStateEnum.ComposeLater, ComposerSentenceListStateEnum.ComposeNow,
```

Comment:
Missing the open state.

From the acceptance criteria:
Apply filters by default upon login to show only sentences with the following statuses: compose later; compose now; needs further review; **open**; ready to compose.